week-1-ts-ecosystem/w01e01-typescript-basics/src/helpers/hasGivenAge.ts

```diff
@@ -0,0 +1,7 @@
+import { User } from "../types";
+
+function hasGivenAge(requiredAge: User["age"]) {
```

Comment:
ciekawe uzależnienie hasGivenAge od samego `User["age"]`. Nie wpadłbym - pewnie walnął tylko klasyczny `:number`

Ale pomysł jest całkiem spoko.
Jeszcze większy miało by to sens - jakbyś podszedł do tematu `feature-like` i podzielił to mniej wiecej tak:
```
user
  | - User.d.ts (tu mieszka model danych)
  | - user.validators.ts (tu mieszkają wszyskie walidatory powiązane z userem jako named exporty)
  |-  user.data.ts (tutaj mamy miejsce na dane)
main.ts
```